app/js/components/orderEntry/ActiveOrders.jsx

```diff
@@ -1,67 +1,139 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import format from 'date-fns/format';
+import PropTypes from 'prop-types';
+import activeOrderAction from '../../actions/activeOrderAction';
+import imageLoder from '../../../img/loading.gif';
+
+export class ActiveOrders extends React.Component {
+  componentDidMount() {
+    this.fetchActiveOrders();
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.tabName !== this.props.tabName) {
+      this.fetchActiveOrders();
+    }
+  }
+
+  fetchActiveOrders() {
+    const { location, careSetting } = this.props;
+
+    const query = new URLSearchParams(location.search);
+    const patientUuid = query.get('patient');
+    const caresettingUuid = careSetting.uuid;
+
+    this.props.activeOrderAction(caresettingUuid, patientUuid);
+  }
+
+  showOrders = activeOrders => activeOrders
+    .map((order) => {
+      const {
+        uuid,
+        dateActivated,
+        autoExpireDate,
+        drug,
+        dose,
+        doseUnits,
+        frequency,
+        route,
+        duration,
+        durationUnits,
+        dosingInstructions,
+        quantity,
+        quantityUnits,
+      } = order;
+
+      return (
+        <tr key={uuid}>
+          <td className="text-center fs-14-px">
+            {format(dateActivated, 'MM/DD/YYYY')} to {format(autoExpireDate, 'MM/DD/YYYY HH:mm')}
+          </td>
+          <td className="text-center fs-14-px">
+            Active
+          </td>
+          <td className="fs-14-px">
+
+            {drug.display}:
```

Comment:
Please fix this table as we discussed. ```.display``` bug.

Reply:
I'm looking into it.

Reply:
did you get this working?

Reply:
Syncing with @geofrocker and the team on this and we are making progress.
Will update this PR with the changes after checking for all possible edge cases.